components/spin/index.tsx

```diff
@@ -118,7 +121,7 @@ export default class Spin extends React.Component<SpinProps, any> {
         <Animate
           {...divProps}
           component="div"
-          className={`${prefixCls}-nested-loading`}
+          className={`${prefixCls}-nested-loading ${wrapperClassName}`}
```

Comment:
I think we can just use [`classNames`](https://github.com/ant-design/ant-design/blob/18da0deb5e279f95371944ee0c41cc7b5a4d7908/components/affix/index.tsx#L4). Because it will not generate extra space, and this means that we don't need to update snapshots.